ironfish/src/rpc/routes/chain/followChainStream.ts

```diff
@@ -78,9 +78,7 @@ routes.register<typeof FollowChainStreamRequestSchema, FollowChainStreamResponse
 
     const send = (block: Block, type: 'connected' | 'disconnected' | 'fork') => {
       const transactions = block.transactions.map((transaction) => ({
-        ...(request.data?.serialized
-          ? { serialized: transaction.serialize().toString('hex') }
-          : {}),
+        serialized: transaction.serialize().toString('hex'),
```

Comment:
does serializing a transaction have a performance impact? is that why this is optional?

Reply:
There are a couple use cases for this endpoint that I know of
1. Standalone wallet + internal wallet both subscribe to a chain stream and need the full serialized transaction
2. Syncer for the block explorer also subscribes to chain stream and does not use the serialized transaction

Adding it as default would increase bandwidth usage for the block explorer syncer but it would also make the endpoints more consistent. We would not run into any bandwidth issues for the syncer if we added the `serialized` field. The only case I can think of that could run into bandwith issues is something on mobile but even a mobile wallet would need the fully serialized transactions. So returning `serialized` every time although not perfectly efficient seems like a fine thing to do here